### ISSUE TITLE
Rename `invokeFunction` and `testInvoke` methods

### ISSUE THREAD

**Issue by devhawk** [CityOfZion/web-wallet#13]
I'm guessing that the names `testInvoke` and `invokeFunction` are for back compat w/ older version of Neo and/or NEON wallet. But it's very confusing that `invokeFunction` wallet command is *NOT* the same as [`invokefunction` RPC method](https://docs.neo.org/docs/en-us/reference/rpc/latest-version/api/invokefunction.html).

Given that the wallet connect infrastructure is so new, is it possible to change the wallet methods to match the rpc method names?

**Comment by lllwvlvwlll** [comment]
The protocol methods were never intended to have a 1-1 mapping to rpc methods since there are features we want to expand which dont require node interfacing.  This was a bit of an oversight.  We are currently discussing how to handle this more cleanly and will most likely need to push the rpc methods down 1 level in the request payload format to accommodate this since the flat structure does not properly reflect this distinction.

**Comment by devhawk** [comment]
For cases where there is no corollary like `signMessage`, that makes perfect sense. But it's very strange that `invokeFunction` wallet method != `invokefunction` RPC

**Comment by melanke** [comment]
I agree to change this method name, it is very different than a invokefunction RPC method and it will receive many upgrades to facilitate the usability, so, yea. What about `publishInvoke`? We could change it on `beta` branch (which is the future v2

**Comment by devhawk** [comment]
> I agree to change this method name, it is very different than a invokefunction RPC method and it will receive many upgrades to facilitate the usability, so, yea. What about publishInvoke? We could change it on beta branch (which is the future v2

Why not just match the RPC method name? The method currently called `testInvoke` would be called `invokeScript` and the method named `invokeFunction` would be `sendRawTransaction`.
